package.json

```diff
@@ -16,6 +16,7 @@
   "author": "Discord Bot List <support@discordbots.org> (https://discordbots.org)",
   "license": "Apache-2.0",
   "dependencies": {
+    "discord.js": "^11.3.0",
```

Comment:
Remove discord.js from dependencies.
Not everyone using this module is using discord.js and installing discord.js just for this module is overkill